templates/etc_systemd_system_postgresql.service.d_custom.conf.j2

```diff
@@ -8,4 +8,8 @@ Group={{ postgresql_service_group }}
 Environment=PGDATA={{ postgresql_conf_directory }}
 
 ExecStartPre=
```

Comment:
Why leave this blank line here... it's a historical typo, right?

Reply:
Blank line deleted.